packages/govern-console/src/containers/CreateDAO/CreateDAO.tsx

```diff
@@ -74,6 +74,23 @@ interface ResultProps {
   postResultActionRoute: string;
 }
 
+interface FormInputs {
+  /**
+   * Use's input
+   */
+  daoName: string;
+  tokenName: string;
+  tokenSymbol: string;
+  tokenAddress: string;
+  isExistingToken: boolean;
+  isUseProxy: boolean;
+  isUseFreeVoting: boolean;
```

Comment:
@Rekard0 

can we have `useProxy` and `useVocdoni` instead of `isUseProxy` and `isUseFreeVoting` ?